OneSignalSDK/onesignal/src/main/java/com/onesignal/PermissionsActivity.java

```diff
@@ -32,13 +32,15 @@
 import android.content.pm.PackageManager;
 import android.os.Build;
 import android.os.Bundle;
+import android.os.Handler;
 import android.support.annotation.NonNull;
 
 import com.onesignal.AndroidSupportV4Compat.ActivityCompat;
 
 public class PermissionsActivity extends Activity {
 
    private static final String TAG = PermissionsActivity.class.getCanonicalName();
+   private static final int DELAY_TIME_CALLBACK_CALL = 500;
    private static final int REQUEST_LOCATION = 2;
```

Comment:
Lets create a `PermissionUtils` eventually and put everything global or constant based in there, just so we keep `PermissionsActivity` clean